english/crazy/crazy.md

```diff
@@ -1,2 +1,7 @@
 Crazy man!
 
+Do you...
+
+[Run](../run/run.md)
+[Hide](..hide/hide-chk.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.